controllers/paymentController.php

```diff
@@ -45,6 +45,11 @@ class WcPagantisGateway extends WC_Payment_Gateway
      */
     private $allowed_currencies;
 
+    /**
+     * @var string
+     */
+    private $paymentProcessingToken;
```

Comment:
Please change it, we told you twice:  
- https://pagantis.slack.com/archives/G017X8BB0C8/p1600860303011700
- https://pagantis.slack.com/archives/G017X8BB0C8/p1600854426008600

Reply:
if the name is identical in both payment methods it create issues so I have renamed it to ` $urlToken` for the normal method and ` $urlToken4x` for the 4x method